site/build_docs.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import os
+import subprocess
+
+from packaging import version
+import git
+
+MINIMUM_VERSION='1.5.0'
```

Comment:
@dvkruchinin Could you change the version to `0.1.11` here? Datumaro has different versions than CVAT.